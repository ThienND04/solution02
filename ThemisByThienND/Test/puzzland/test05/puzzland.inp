43
4 6
I S A V 
I V
A V
S V
I S
S A
I A
4 4
Y B R V 
B V
R V
Y R
B R
7 16
D V W S K U L 
V K
W K
W S
V U
S L
S K
D K
W U
V L
K U
U L
D V
V W
D S
D U
D W
7 21
D A L G F Y W 
L F
F W
A L
D W
D L
G F
G W
L W
L G
D Y
D A
G Y
D G
A G
Y W
D F
A Y
F Y
A W
L Y
A F
4 5
F G W T 
G T
F T
G W
W T
F W
9 19
C D A Q H T S G L 
H T
D H
A L
A T
C L
A H
D G
A Q
S G
C A
D L
C Q
H L
G L
Q T
T S
Q G
C G
D A
6 14
U L N I T V 
U L
U V
N V
I T
I V
T V
U I
N T
L I
L T
L N
U T
L V
U N
10 38
O Q X J T C N R H D 
X C
T C
C D
T R
Q N
X J
T D
C H
C R
O X
Q J
O H
N R
X H
J H
N D
Q H
X T
N H
O T
Q C
J N
O C
R D
O J
H D
J C
J T
O D
Q D
O R
T N
X R
J D
Q T
C N
R H
J R
6 15
I V G U S D 
S D
G U
V D
U S
V G
G D
I S
U D
I D
I V
G S
I U
I G
V S
V U
7 13
C U P L O A K 
U K
P L
P A
P K
C A
O K
C O
C K
A K
U P
O A
C U
C L
6 14
Y C J F W E 
Y J
C E
J E
F E
Y C
W E
C F
J W
F W
Y E
C W
Y W
Y F
C J
4 4
D K P Z 
D K
D P
P Z
K P
8 20
I V D R G B E H 
B H
D H
I R
R B
R G
V G
I H
I V
D G
R H
E H
G B
G E
V H
V E
I E
D R
D B
B E
I B
6 10
W F R K A Z 
K A
W R
W A
R Z
F A
A Z
F K
F R
W F
W K
6 11
M P V S Y T 
S T
M V
M T
P V
V T
Y T
V Y
M P
S Y
V S
P T
10 31
X W P J D A L G H T 
W P
W D
X L
W H
X W
D A
A G
X P
A T
P H
G H
X T
G T
D L
X H
L G
J A
X A
P G
J G
P T
P A
X D
W L
H T
J L
W J
P L
J T
W G
X G
10 31
N F D A Q P M Z G W 
D Q
N D
F M
D G
A Z
A M
P G
P M
F G
N P
M W
P Z
N Z
A W
Q G
M G
D P
D Z
Q Z
P W
N F
Z G
D M
D A
N A
F A
A Q
F W
N W
Z W
F Q
10 40
J W Q N A H L X I Z 
J A
W X
H I
W L
Q H
W Q
I Z
L X
J Q
Q Z
X I
J Z
A L
H Z
A Z
Q X
N X
H L
A I
Q N
J L
Q I
Q L
N Z
L I
J I
W H
W I
W N
L Z
N I
N A
J W
W A
A X
J H
A H
N H
W Z
J X
7 14
U P Y D X Q Z 
P Q
U Y
P Y
Y Z
P X
X Z
Y D
U P
D X
Y Q
D Q
U Q
U D
P Z
4 6
R L K Q 
L Q
R Q
K Q
R K
R L
L K
4 6
J D G S 
J S
D S
J D
G S
J G
D G
5 8
F I Z D P 
I D
F I
F Z
D P
I P
I Z
Z D
F D
6 13
X U I G R Q 
U Q
I R
R Q
U R
X G
X U
G R
U I
X I
I G
X Q
X R
U G
8 16
E C W T A G J B 
A B
T G
C G
J B
C J
W J
E W
C W
C A
A J
E T
W A
W B
C T
T J
E C
9 32
V J A D P K I C E 
V I
A K
I E
J E
P E
V E
J D
P I
D E
K I
V P
C E
K E
P K
I C
P C
A I
D K
J A
D I
V K
D P
J I
K C
J C
A P
V A
J K
A C
A E
V C
V J
10 41
P G V F O W N J C L 
C L
G F
P G
V W
N J
V N
O L
V O
G J
F W
F J
O C
W N
G C
F O
P O
J L
O J
P W
P V
P J
G W
W J
G N
F N
P C
N C
F C
J C
G L
N L
F L
V C
P N
V L
P L
V J
V F
G V
W L
W C
8 23
A Q R J B Y I H 
A Q
A J
J H
A B
Q H
B H
Q R
A R
R Y
J Y
R H
Y H
B I
A Y
B Y
Q Y
A H
A I
Q J
Q B
J I
I H
J B
10 41
W E S G F O M N Y U 
G U
F N
F M
E O
E Y
F O
W O
F Y
S N
S M
E U
W U
M N
S Y
O N
E N
G F
O Y
W G
W F
W S
G N
F U
W Y
W M
E M
S O
S U
N U
O M
Y U
G Y
S G
E G
M U
O U
N Y
E S
S F
W E
E F
9 33
J U A G H F Y V Q 
J H
A Y
G F
J U
F V
U Y
V Q
G Y
H V
J F
J G
U H
U A
U Q
H Y
G V
U F
F Y
H F
A V
Y Q
G Q
A G
J V
U V
U G
Y V
A F
J Y
A Q
J Q
A H
G H
5 7
M Z D E Y 
M E
D E
M Y
M D
M Z
Z D
Z Y
7 18
J P U K R E X 
K X
R X
E X
R E
P X
J X
J E
J R
P R
K E
U R
K R
J U
U K
J P
U X
P E
U E
9 23
P D O H N Z K T X 
P K
N T
P H
D K
K T
Z X
K X
D X
P O
D T
H T
H K
O K
N K
N Z
P Z
N X
D O
P X
Z K
D N
P T
H Z
5 7
C G O E I 
G I
O I
C G
G E
C O
C E
O E
5 7
K Y H G W 
G W
Y W
K Y
Y H
K W
K H
H G
9 22
U W H E S K X V O 
K O
V O
S V
U O
U S
H E
U X
S X
E K
H X
K V
E O
U V
W H
X O
E S
H S
U E
U H
U K
W X
K X
6 8
L O U N T Z 
O Z
L T
L Z
N Z
N T
U N
T Z
U T
7 13
U I K V D M A 
D A
V A
U M
U K
I D
K D
I K
M A
U D
U I
D M
U A
I A
6 12
W T S J P R 
J P
S J
W S
T J
T P
W R
W J
W P
S R
J R
T S
P R
10 29
X T C G Y D A W B K 
A W
C Y
Y D
T B
C B
C G
T C
D W
X Y
Y W
Y K
X W
T W
C W
T D
G A
G K
X K
Y A
X B
W K
C D
D A
C A
X A
G D
W B
T K
T A
6 12
I E Y Z H G 
Z H
E H
E Y
I E
Y G
Z G
Y H
I G
E Z
E G
I Z
Y Z
7 13
O X M T F L D 
M T
M F
T D
T F
O D
X F
O L
M L
T L
O M
M D
X T
L D
7 13
X O K B C G W 
K B
C W
O W
G W
K G
X B
B C
X C
X K
B W
O C
B G
C G
7 17
Z D E X W U Y 
Z Y
Z D
Z W
Z X
X W
X Y
E X
D Y
Z U
D W
D E
E U
Z E
D X
W U
E Y
E W
