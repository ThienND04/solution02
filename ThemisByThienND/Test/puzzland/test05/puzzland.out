IASV
YRBV
DKSWUVL
DAFGLYW
FWGT
CAQGSTHDL
UILNTV
OCHJNQTXRD
IGSUVD
IMPOSSIBLE
YCFWJE
DKPZ
IBDRGEVH
WAKFRZ
MPVSYT
XADLGHPWJT
NADGFQZPMW
JAHILQNXWZ
UDQYPXZ
RKLQ
JDGS
FDZIP
XGIRUQ
ECGTJAWB
VACIDJKPE
PCFGJNWVOL
ABIJQRYH
WEFGNMOSYU
JFAGHUVYQ
MEDZY
JEKURPX
PHKODTNZX
CGEOI
KYHGW
UEHWXSVKO
IMPOSSIBLE
IMPOSSIBLE
WJPTSR
XACBTWYDGK
IEHYZG
OLMFXTD
XBKGCOW
ZDEUWXY
